commit 3a35c27ac68cea19c252e127ec61099648eb4870
Author: Randy Dunlap <randy.dunlap@oracle.com>
Date:   Wed Aug 6 10:08:56 2008 -0700

    docbook: fix v4l fatal filename error
    
    docproc: /var/linsrc/lin2627-rc2/drivers/media/video/videodev.c: No such file or directory
    make[1]: *** [Documentation/DocBook/videobook.xml] Error 1
    
    Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
    cc: mchehab@infradead.org
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

diff --git a/Documentation/DocBook/videobook.tmpl b/Documentation/DocBook/videobook.tmpl
index 8981779..0bc2594 100644
--- a/Documentation/DocBook/videobook.tmpl
+++ b/Documentation/DocBook/videobook.tmpl
@@ -1648,7 +1648,7 @@ static struct video_buffer capture_fb;
 
   <chapter id="pubfunctions">
      <title>Public Functions Provided</title>
-!Edrivers/media/video/videodev.c
+!Edrivers/media/video/v4l2-dev.c
   </chapter>
 
 </book>
